e2e/src/utils.ts

```diff
@@ -157,7 +157,7 @@ export const utils = {
       for (const table of tables) {
         if (table === 'system_metadata') {
           // prevent reverse geocoder from being re-initialized
-          sql.push(`DELETE FROM "system_metadata" where "key" != 'reverse-geocoding-state';`);
+          sql.push(`DELETE FROM "system_metadata" where "key" NOT IN ('reverse-geocoding-state', 'system-flags');`);
```

Comment:
nits: comment is probably redundant + will not clearning the system flags potentially cause issues elsewhere/later?

Reply:
It might yeah. It's the easiest option right now though. In the future we might be able to wait for microservices to fully start up with a custom implementation.